### ISSUE TITLE
[BUG] renderSourceTypeIcon violates jsx-a11y/alt-text

### ISSUE THREAD

**Issue by harupy** [mlflow/mlflow#2051]
<img width="634" alt="Screen Shot 2019-11-07 at 12 55 59" src="https://user-images.githubusercontent.com/17039389/68359097-15a65f80-015e-11ea-90be-e78f8a417225.png">

```javascript
  static renderSourceTypeIcon(sourceType) {
    const imageStyle = {
      height: '20px',
      position: 'relative',
      top: '-1px',
      marginRight: '2px',
    };
    if (sourceType === "NOTEBOOK") {
      return <img title="Notebook" style={imageStyle} src={notebookSvg} />;
    } else if (sourceType === "LOCAL") {
      return <img title="Local Source" style={imageStyle} src={laptopSvg} />;
    } else if (sourceType === "PROJECT") {
      return <img title="Project" style={imageStyle} src={projectSvg} />;
    }
    return <img style={imageStyle} src={emptySvg} />;
  }
```

https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/utils/Utils.js#L247-L262

**Comment by harupy** [comment]
I'll open a PR.

**Comment by harupy** [comment]
@smurching @Zangr 
Should I disable ` jsx-a11y/alt-text` or add `alt` to the image tags?

**Comment by harupy** [comment]
resolved via #2083
